Project.toml

```diff
@@ -1,22 +1,25 @@
 name = "DiffRules"
 uuid = "b552c78f-8df3-52c6-915a-8e097449b14b"
-version = "1.9.0"
+version = "1.9.1"
```

Comment:
Is this a breaking change @devmotion ?

Reply:
It shouldn't, it should just fix `Float32` etc. issues. Did I miss anything breaking?

Reply:
No I agree, but the discussion regarding including a dependency made me worried and so I was thinking maybe we should make it breaking just to be on the safe-side. But _I_ would personally decide to make it non-breaking; just wanted to check with one of you.